e2e/staticpvc.go

```diff
@@ -189,6 +201,39 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock, checkI
 	return err
 }
 
+func validateRBDStaticResize(
+	f *framework.Framework,
+	app *v1.Pod,
+	appOpt *metav1.ListOptions,
+	pvc *v1.PersistentVolumeClaim,
+	rbdImageName string) error {
+	// resize rbd image
+	size := "8Gi"
```

Comment:
may be declare or use a var for this size?

Reply:
isn't it already a short variable declaration? did you mean something else?

Reply:
declared a new global variable